system/lib/wasmfs/memory_file.h

```diff
@@ -17,12 +17,10 @@ namespace wasmfs {
 // This class describes a file that lives in Wasm Memory.
```

Comment:
As this file now contains MemoryFile and MemoryDirectory, perhaps let's rename it to `memory_backend.h` or such?

Reply:
This has landed separately in #16286